### ISSUE TITLE
MCP Email Copy Update #3

### ISSUE THREAD

**Issue by kjsuarez** [department-of-veterans-affairs/va.gov-team#75555]
Once #75545 is complete we can update the MCP notification copy to include the copay outstanding balance.
[mcp_notification_copy.docx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNHc3QXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4f7305ec9165d5bec1a070b9f41a5c42cf90d7ca/Jan_30_MCP_Notification_Update%20%281%29.docx)

- [ ] Update `app/sidekiq/copay_notifications/new_statement_notification_job.rb` to include outstanding balance in the personalization hash
- [ ] Update our VaNotify template (via the vpn) to use updated copy

**Comment by denisecoveyduc** [comment]
Hi @kjsuarez Here is the updated copy for the V3 version of the email. As soon as VBS  has added the copay amount outstanding to the data we can move forward in implementing this change. Adding the facility and the $ value will add some much needed clarity that Veterans are looking for. 

@Tom-Davis @HeatherRienks Note above :) I know this version is being used in the user research and we may identify some feedback out of that but I'd rather move forward with this version and then do a subsequent update if needed after the research is finalized. 

[MCP Notification Updates_V3.docx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeEpoQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--360739e29a2cc13c395c74f130437e11690be1d9/MCP%20Notification%20Updates_V3.docx)

**Comment by HeatherRienks** [comment]
We are still awaiting feedback from VBS

**Comment by denisecoveyduc** [comment]
This ticket will be blocked until a few decisions (listed below for clarity) have been reached on the VA side. Each of these discussions are ongoing at the moment and I will provide an update once this has been unblocked. 

These blockers include:

- [ ] Alignment on copy of email
- [ ] Coordination with the VA Notify team to establish notification preferences for Veterans
- [ ] Business consensus to timing of restart (before or after VHA Payment History is live in the debt portal)

@HeatherRienks @Tom-Davis Given the extent of these blockers I do think it is best we move this to blocked as the team can not move this forward at this time.

**Comment by HeatherRienks** [comment]
This work continues to remain blocked by items noted above.

**Comment by digitaldrk** [comment]
I'm not really sure what needs to happen on this one. Seems like it's been around for a while.

**Comment by Tom-Davis** [comment]
@digitaldrk yes it has - I believe the second blocker in the list has been completed but need to discuss with @denisecoveyduc on the first and last item. I added it to out discussion points. If it is going to be much longer we might consider moving this to the backlog for future consideration.

**Comment by Tom-Davis** [comment]
Moving this to backlog as the blocker will not be resolved in the near future.

**Comment by denisecoveyduc** [comment]
@Tom-Davis This one is related to re-establishing the copay email notifications that are currently turned off. I will change the parent issue and we can revisit once we pick that work back up.
